design/admin/templates/parts/newsletter/menu.tpl

```diff
@@ -4,7 +4,8 @@
 
         {include uri='design:parts/newsletter/newsletter_menu.tpl'}
 
-
+        {def $adminMenu= fetch( 'user', 'has_access_to', hash( 'module',   'newsletter', 'function', 'admin'  ) ) }      
```

Comment:
Like you define a variable in the template, please add a {undef} at the end of it.